airbyte-webapp/src/components/ToolTip/TooltipTable.tsx

```diff
@@ -0,0 +1,24 @@
+import { useTooltipContext } from "./context";
+import styles from "./TooltipTable.module.scss";
+
+interface TooltipTableProps {
+  rows: React.ReactNode[][];
+}
+
+export const TooltipTable: React.VFC<TooltipTableProps> = ({ rows }) => {
+  const { theme } = useTooltipContext();
```

Comment:
For my own learning, could you briefly explain why you chose to use a context here? Is this because the `theme` prop could not be passed directly to `TooltipTable`, because that sub-component is not always present?

Reply:
I did this in order to understand the context of the tooltip without having to pass the props twice. Otherwise in the component that implements you you'd have to pass the same theme:

```tsx
<Tooltip theme="light">
  <TooltipTable theme="light" />
</Tooltip>

// Nothing stopping you from doing:
<Tooltip theme="light">
  <TooltipTable /> // If you don't pass the theme and have no context, the theme of the table will be "dark"
</Tooltip>
```

Reply:
Ah okay, so this is basically forcing the TooltipTable to match the same theme as its parent Tooltip through the use of context, removing that burden from the developer. That is cool!